resources/views/components/translatable-input.blade.php

```diff
@@ -0,0 +1,21 @@
+<div class="field @error($name . '.' . locale()) field--error @enderror">
+    <x-hearth-label :for="$name . '_' . locale()" :value="__(':label (:locale)', ['label' => $label, 'locale' => get_locale_name(locale())])" />
+    <x-hearth-input type="text" :id="$name . '_' . locale()" :name="$name . '[' . locale() . ']'" :value="old($name . '.' . locale(), $model ? $model->getTranslation($name, locale()) : '')" />
```

Comment:
I wonder if long string names like this would be easier to read using php string templates. Is that possible in the blade template?

Reply:
It appears that in component attributes you can't use string interpolation. At least, I've had no luck doing so…

Reply:
Thanks for checking.